### DISCUSSION TITLE
Twilio for Calling / PBX

### DISCUSSION THREAD

**Discussion by deepti4666** [YetiForceCompany/YetiForceCRM#14261] [How to]
How can I add Twilio for calling a contact from Yetiforce?

**Comment by mariuszkrzaczkowski** [comment]
look here 
https://github.com/YetiForceCompany/YetiForceCRM/tree/developer/app/Integrations/Pbx

**Reply by deepti4666** [reply]
Thank you - am not sure how to implement this
